openstack/identity/v3/federation/metadata/requests.go

```diff
@@ -0,0 +1,34 @@
+package metadata
+
+import golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+
+type ImportOptsBuilder interface {
+	ToMetadataImportMap() (map[string]interface{}, error)
+}
+
+type ImportOpts struct {
+	XAccountType string `json:"xaccount_type"`
```

Comment:
omitempty?

Reply:
No, it's required from documentation point of view, but is empty I'm most cases